ietf/doc/models.py

```diff
@@ -617,7 +617,7 @@ def pdfized(self):
         if text:
             stylesheets.append(finders.find("ietf/css/document_html_txt.css"))
         else:
-            text = self.htmlized()
+            text = f'<div class="rfcmarkup">{self.htmlized()}</div>'
```

Comment:
maintainability question: why didn't this go into the htmlized function? If this is really only pdf specific, then maybe the class should reflect that so it doesn't get applied to other things by future people who didn't understand the intent?

Reply:
It could go there. I'll move it.